readthedocs/api/mixins.py

```diff
@@ -80,7 +80,10 @@ def unresolved_url(self):
         url = self.request.GET.get("url")
         if not url:
             return None
-        return unresolve(url)
+        try:
+            return unresolve(url)
+        except UnresolverError:
+            return None
```

Comment:
These ones could have a comment explaining why we are ignoring these exceptions.